### PR TITLE
chore: remove deprecated methods from system test

### PR DESCRIPTION
**Pull request by athakor**

### CODE REVIEW

File: google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java [hunk 1]
```diff
@@ -3046,13 +3048,16 @@ public void testEnableAndDisableBucketPolicyOnlyOnExistingBucket() throws Except
       Bucket remoteBucket =
           storage.get(bpoBucket, Storage.BucketGetOption.fields(BucketField.IAMCONFIGURATION));
 
-      assertTrue(remoteBucket.getIamConfiguration().isBucketPolicyOnlyEnabled());
-      assertNotNull(remoteBucket.getIamConfiguration().getBucketPolicyOnlyLockedTime());
+      assertTrue(remoteBucket.getIamConfiguration().isUniformBucketLevelAccessEnabled());
+      assertNotNull(remoteBucket.getIamConfiguration().isUniformBucketLevelAccessEnabled());
```
Comment (dmitry-fa) [inline google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java]: isUniformBucketLevelAccessEnabled is used twice.
please replace with uniformBucketLevelAccessLockedTime

**Review by dmitry-fa — CHANGES_REQUESTED** [review]

File: google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java (hunk 1 above)
Comment (dmitry-fa) [reply google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java]: other looks good.

Comment (athakor) [reply google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java]: > isUniformBucketLevelAccessEnabled is used twice.
> please replace with uniformBucketLevelAccessLockedTime

done. thanks for catching this

Comment (athakor) [reply google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java]: @dmitry-fa PTAL

**Review by rahulKQL — APPROVED** [review]

**Review by dmitry-fa — APPROVED** [review]
Looks good

**Pull request closed without merging**